CI-CD/README.md

```diff
@@ -39,11 +39,11 @@ See also: [Cache the Latest Version of the Unified Agent](../Scripts/README.md#c
 ### GitHub Actions
 
 ```
-- name: 'Upload WhiteSource folder'
+- name: 'Upload Mend folder'
   uses: actions/upload-artifact@v2
   with:
-    name: WhiteSource
-    path: whitesource
+    name: Mend
+    path: Mend
```

Comment:
Same as above, hard-coded or arbitrary name in this context?

Reply:
@lukebrogan-mend this should remain /whitesource as it is the folder for the unified agent scan...which I believe is still whitesource